OpenEdXMobile/src/main/java/org/edx/mobile/util/Version.java

```diff
@@ -142,4 +142,19 @@ public VersionParseException(final int location) {
             super("Token couldn't be parsed as a valid number.", location);
         }
     }
+
+    /**
+     * Compares this version to the specified version to determine if minor versions
+     * difference between both is greater than the or equal to the specified value.
```

Comment:
The grammar of your description is wrong, also the function you mentioned compares 2 versions keeping in account their major, minor and patch version. Which is not exactly the case in this function.